peps/pep-0736.rst

```diff
@@ -238,6 +230,44 @@ for the ``f(x=)`` form for the following reasons:
 * `A poll of Python developers <https://discuss.python.org/t/syntactic-sugar-to-encourage-use-of-named-arguments/36217/130>`__
   indicates that this is the most popular syntax among those proposed.
 
+How to Teach This
+=================
+
+Keen Python developers will likely hear about this feature through typical
+information channels, such as newsboards, social media, mailing lists, online
+forums, or word of mouth. Many more will encounter this feature while reading
+code and noting the omission of the value in a keyword argument at invocation,
+violating their expectations. We should ensure such developers have easy access
+to documentation that explains the semantics of this feature and that this
+documentation is easy to find when searching. For example, the
+`Python Glossary <https://docs.python.org/3/glossary.html#term-argument>`__ and
+`Tutorial <https://docs.python.org/3/tutorial/controlflow.html#keyword-arguments>`__
+may be updated accordingly and reasonable keywords may be used to help with
+search discoverability.
+`A StackOverflow question <https://stackoverflow.blog/2011/07/01/its-ok-to-ask-and-answer-your-own-questions/>`__
+could be written to help explain this feature to those searching for an
+explanation.
+
+To ease the communication of and search for this feature, it may also be
+valuable to provide this feature with a name, such as 'keyword argument
+shorthand'.
+
+Newer Python programmers may learn about this feature as an optional abbreviated
+syntax where keyword arguments are taught, such as in tutorials or classrooms.
+A teacher may explain this feature as, "where you see an argument followed by an
+equals sign, such as ``f(x=)``, this represents a keyword argument where the
+name of the argument and its value are the same. This can be written
+equivalently in the expanded notation, ``f(x=x)``." Depending on a student's
+background, a teacher might further compare this to equivalent syntax in other
+languages or Python's f-string syntax ``f"{x=}"``.
+
+To understand this, a student of Python would need to be familiar with the
+basics of functions in addition to the existing keyword argument syntax.
+Given that this feature is a very simple syntactic sugar, it is reasonable that
```

Comment:
```suggestion
Given that this feature is syntactic sugar, it is reasonable that
```

Reply:
I'm not sure I fully agree here. If the syntactic sugar were complex, I can see how this expectation may be unreasonable. E.g. the use of `super()` in `__init__` (PEP 3135)

Reply:
A fair perspective @joshuabambrick, and thanks for making the other changes. I was hoping to avoid the phrase "very simple" since in teaching it can be somewhat dismissive if someone doesn't understand something. I would be fine with removing "very" or replacing with "uncomplicated" or "straightforward".

Reply:
Replaced with 'relatively straightforward'. Let me know of any objections